specification/columns/availabilityzone.md

```diff
@@ -2,6 +2,16 @@
 
 An [*availability zone*](#glossary:availability-zone) is a provider-assigned identifier for a physically separated and isolated area within a Region that provides high availability and fault tolerance. Availability Zone is commonly used for scenarios like analyzing cross-zone data transfer usage and the corresponding cost based on where [*resources*](#glossary:resource) are deployed.
 
+---
+The AvailabilityZone column adheres to the following requirements:
+
+* AvailabilityZone is RECOMMENDED to be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider supports deploying resources or services within an *availability zone*.
+* If present, the column MUST conform to the following additional requirements:
+  * AvailabilityZone MUST be of type String.
+  * AvailabilityZone MUST conform to [String Handling](#stringhandling) requirements.
```

Comment:
```suggestion

```

Discussed in Dec 10 TF1.  Propose postponing adding such requirements until lower-hanging fruit of consistency is collected.

Reply:
Talking to Irena - she suggested postponing this due to provider conformance concerns.

Reply:
I suggest we discuss this in the TF-1 meeting. This [sheet](https://docs.google.com/spreadsheets/d/1IG8WfRag_Sy5AOZpfUdiaOMmrA_1yJndAMy6ywYSOmg/edit?gid=1714226587#gid=1714226587) can serve as the basis for our discussion.